custom/icds_reports/static/js/directives/info-message/info-message.directive.js

```diff
@@ -6,8 +6,11 @@ function InfoMessageController($location) {
     var vm = this;
     vm.previousToPreviousMonth = null;
     vm.previousMonth = null;
+    vm.previousYear = null;
```

Comment:
these variable names are quite misleading. will propose new ones. this one I think should be `yearOfPreviousMonth`

Reply:
done